<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fontpicker</class>
 <widget class="QWidget" name="fontpicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="textScroll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1080</width>
     <height>530</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QFrame" name="toolsArea">
   <property name="geometry">
    <rect>
     <x>-260</x>
     <y>0</y>
     <width>1280</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="textColor">
    <property name="geometry">
     <rect>
      <x>1075</x>
      <y>35</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="textBackgroundColor">
    <property name="geometry">
     <rect>
      <x>1125</x>
      <y>35</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetSettings">
    <property name="geometry">
     <rect>
      <x>1175</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="searchIconBackground">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>30</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="fontApply">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>335</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 0, 0, .6);
padding-left: 12px;
padding-right: 12px;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>30</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 0, 0, .6);
padding-left: 12px;
padding-right: 12px;
</string>
    </property>
   </widget>
   <widget class="QSlider" name="textSizeChanger">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>textColor</zorder>
   <zorder>textBackgroundColor</zorder>
   <zorder>resetSettings</zorder>
   <zorder>fontApply</zorder>
   <zorder>search</zorder>
   <zorder>textSizeChanger</zorder>
   <zorder>searchIconBackground</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
